docs/pages/access-controls/sso/adfs.mdx

```diff
@@ -201,3 +201,12 @@ automatically in a browser.
 ## Troubleshooting
 
 (!docs/pages/includes/sso/loginerrortroubleshooting.mdx!)
+
+## Next steps
+
+In the Teleport role we illustrated in this guide, `external` traits
+ are replaced with values from the single sign-on provider that the user
+used to authenticate to Teleport. For full details on how traits
+work in Teleport roles, see the  [Teleport Access Controls
+Reference](docs/pages/access-controls/reference.mdx#roles).
```

Comment:
Is there a reason we didn't make this a include?

Reply:
I found myself editing the text slightly to make the instructions fit into each guide, which didn't really lend itself to a partial. If we end up using one variety of these instructions a lot, though, we can definitely extract these into a partial.